### ISSUE TITLE
Add more domain type

### ISSUE THREAD

**Issue by zer09** [CaptainCodeman/clean-go#5]
I am trying to understand how to implement clean architecture.

Is there a chance that you add another domain example not just for Greeting domain?

Thanks

**Comment by CaptainCodeman** [comment]
No, sorry, IMO there's enough there to hopefully demonstrate the concept and the architecture while keeping things simple and undrstandable  - applying it to your own domain is part of developing your app. That's the bit you have to do :)

**Comment by zer09** [comment]
Thanks for the reply, I want to close this but can I ask one more question?

I want to know if I understand this right, does the "providers" is the "Frameworks and Drivers" and the "adapters/web" is the "interface adapters" on the model chart?

**Comment by CaptainCodeman** [comment]
Sorry, I don't understand the question.
